### ISSUE TITLE
Moralis.onAccountChanged is not a function

### ISSUE THREAD

**Issue by socheat-leang** [ethereum-boilerplate/ethereum-react-native-boilerplate#46]
I have cloned the project and done the yarn command successfully in order to load node_modules

Anyway, When I run the project, I met the error below.
`TypeError: Moralis.onAccountChanged is not a function. (In 'Moralis.onAccountChanged(function (address) {
        setWalletAddress(address[0]);
      })', 'Moralis.onAccountChanged' is undefined)`

When I block this code, it can't make connect to wallet successfully. 

The environment, I use the default from this boilerplate. 

Do you have any suggestions or I miss some part? Please guide me in advance.

Thanks

**Comment by alexnguyennz** [comment]
Hi can you try changing that to `Moralis.onAccountsChanged` in the file at `frontend/providers/MoralisDappProvider/MoralisDappProvider.js`.

**Comment by ohmygaugh-crypto** [comment]
> Hi can you try changing that to `Moralis.onAccountsChanged` in the file at `frontend/providers/MoralisDappProvider/MoralisDappProvider.js`.

Yup, I had the same issue and it was resolved with the suggestion above!

Silly typo

**Comment by snehal-delixus** [comment]
Hi have changed to  Moralis.onAccountsChanged in the file at frontend/providers/MoralisDappProvider/MoralisDappProvider.js  
but still i am getting error 404 on https://walletconnect.com/wallets and the popup with all the wallets is not shown

**Comment by alexnguyennz** [comment]
@snehal-delixus  Hi can you post on the Moralis forum in this thread: https://forum.moralis.io/t/ethereum-react-native-boilerplate-questions/4511

**Comment by BasitAlphaA** [comment]
guys i have the same error as mentioned above
TypeError: Moralis.onChainChanged is not a function

but i cant understand where this file exists as @alexnguyennz  mentioned.
@alexnguyennz  can you guide me on this issue, it will be a grate help to me. THANKS

**Comment by alexnguyennz** [comment]
Hey your error is slightly different - in your terminal, it should show you the file and line where that code is being run. You can also do a project search with your IDE or code editor.
